data_integration/pipelines.py

```diff
@@ -81,7 +81,7 @@ def html_doc_items(self) -> [(str, str)]:
 
 
 class Task(Node):
-    def __init__(self, id: str, description: str, commands: [Command] = None, max_retries: int = 0) -> None:
+    def __init__(self, id: str, description: str, commands: [Command] = None, max_retries: int = None) -> None:
         super().__init__(id, description)
         self.commands = []
         self.max_retries = max_retries
```

Comment:
What is shown in the ui in case this is None?

Reply:
nothing, because this line: https://github.com/mara/data-integration/blob/master/data_integration/ui/node_page.py#L88